crates/nu-command/src/strings/encode_decode/encode_base64.rs

```diff
@@ -18,6 +18,14 @@ impl Command for EncodeBase64 {
             .input_output_types(vec![
                 (Type::String, Type::String),
                 (Type::Binary, Type::String),
+                (
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::String)),
+                ),
+                (
+                    Type::List(Box::new(Type::Binary)),
+                    Type::List(Box::new(Type::String)),
+                ),
```

Comment:
Should we relax for heterogeneous list here?

Reply:
do you mean something like
```rust
(
    Type::List(Box::new(Type::Any)),
    Type::List(Box::new(Type::String)),
)
```
?

Reply:
Yup. Done